care/facility/api/viewsets/asset.py

```diff
@@ -98,6 +98,12 @@ class AssetFilter(filters.FilterSet):
     qr_code_id = filters.CharFilter(field_name="qr_code_id", lookup_expr="icontains")
 
 
+class AssetPublicViewSet(RetrieveModelMixin, GenericViewSet):
```

Comment:
Let's also add caching to this endpoint. cache it with a timeout of a day or so, invalidate cache with the asset save method

Reply:
Check the geo api's for ref

Reply:
@vigneshhari What do you mean by the "Geo api"? 😅

Reply:
Added Caching (details below)

Reply:
geo api means lsg, state, dist api's